### DISCUSSION TITLE
{CentOS-7-x64-minimal +headless VPS} I disabled the IPv6 for security and privacy and the WireGuard stopped functioning, uninstall and reinstalled, but WireGuard not working unless I turn IPv6 On, please help??

### DISCUSSION THREAD

**Discussion by perfectpeach** [angristan/wireguard-install#208] [Help]
OS: CenOS-7-x65-minimal +headless
Applications installed on OS: WireGuard, OpenVPN, QBitTorrent(headless), CSF

Hiya,

Wireguard was working until I decided to turn off the IPv6 for security and privacy reasons, as I have done the below:
In the file '/etc/sysctl.conf' I added te below:
#---start
net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
#---end

Then I restarted the IPv6 service. And I restarted the VPS. The WireGuard stopped functioning with the error below:
https://prnt.sc/10apov9

Only WireGuard is not working with IPv6 disabled.

- May OP or someone knowledgable please help me? @randomshell ?
- May please @angristan fix this issue to help us use this WireGuard installer with IPv6 disabled?

Tnx and best of luck

**Comment by angristan** [comment]
Remove the IPv6 from here:

```
root@wg:~# cat /etc/wireguard/wg0.conf
[Interface]
Address = 10.66.66.1/24,fd42:42:42::1/64
```
